### ISSUE TITLE
Hengshuang new result

### ISSUE THREAD

**Issue by yuchenlichuck** [qq456cvb/Point-Transformers#13]
```
batch_size: 32
epoch: 500
learning_rate: 0.0005
gpu: 0
num_point: 1024
optimizer: Adam
weight_decay: 0.0001
normal: true
model:
  nneighbor: 16
  nblocks: 4
  transformer_dim: 512
  name: Hengshuang
```
Best Instance Accuracy: 0.917067, Class Accuracy: 0.893971

**Comment by qq456cvb** [comment]
Thanks for your contribution! I will update the result.

**Comment by ardianumam** [comment]
Hi @qq456cvb , @yuchenlichuck ,
In the paper, it is written that mean ACC = 90.6 and overall ACC = 93.7. Does the number above (91.7067) correspond to overall ACC? And if yes, the number is pretty high, even DGCNN achieves higher (92.2).

Thanks.

**Comment by yuchenlichuck** [comment]
Yes, and for the paper result, the author still hasn't open-sourced the code yet, so we haven't achieved that result.
